my-app/package.json

```diff
@@ -1,6 +1,9 @@
 {
-  "main": "node_modules/expo/AppEntry.js",
+  "name": "my-app",
+  "version": "1.0.0",
+  "main": "__generated__/AppEntry.js",
```

Comment:
You can change this path to whatever you'd like. In the `expo-yarn-workspaces`, `__generated__/AppEntry.js` is used. But you can also do something like [`.expo/AppEntry.js` as you can see here](https://github.com/byCedric/use-expo/blob/master/example/package.json#L8). Any path that you declare here is used after running `expo-yarn-workspaces postinstall`.